input/blog/2017-09-13-cake-v0.22.0-released.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cake v0.22.0 released
+category: Release Notes
+author: mholo65
+---
+
+Version 0.22.0 of Cake has been released.
+
+So, let's be clear, this release will break things - but it is all intentional!
+
+This release was initially about updating Roslyn, unifying scripting engines, and targeting .NET 4.6. But one thing led to another which resulted in better support for tool and addin versioning when using preprocessor directives, and an optional in-process NuGet client.
+
+Updating Roslyn and unifying scripting engines has been a long-awaited improvement for many of you. This brings you C# 7 features (without need for `--experimental` switch), debugging with Mono (as we completely removed the old Mono scripting engine), and much, much more. It also allows us to ship Roslyn with Cake, so there's no need for Cake to actually download the Roslyn anymore. In order for us to update Roslyn, we also needed to change target framework to .NET 4.6.
```

Comment:
Is it worth adding a (very) short sentence/phrase explaining that the change to .NET 4.6 shouldn't really affect most users?